packages/cli/src/lib/cmds/updateConfig.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Config, Log} from '@bubblewrap/core';
+import {ParsedArgs} from 'minimist';
+import {join} from 'path';
+import {homedir} from 'os';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+
+const CONFIG_FILE_PATH = join(join(homedir(), '.bubblewrap-config/bubblewrap-config.json'));
```

Comment:
We already have this value defined in `config.ts`, we should export it use that instead otherwise we're in danger of someone changing one and not the other and things going wrong.

Reply:
Also, @andreban made a change to where the config file is stored (from `~/.bubblewrap-config/bubblewrap-config.json` to just `~/.bubblewrap/config.json`). You'll need to pull from master and merge/rebase your branch to get his changes.

https://github.com/GoogleChromeLabs/bubblewrap/commit/e38ea223a5f2b2b6a29ddc875502fb5275116460